rflx/model/message.py

```diff
@@ -1180,6 +1181,47 @@ def __prove_field_positions(self) -> None:
                         )
                         return
 
+    def __prove_message_size(self) -> None:
+        """Prove that all message variants have a size that is a multiple of 8 bit."""
```

Comment:
What are message variants? Shouldn't that be "all message paths".

Reply:
For me a message has multiple variants which are described by the paths through the message model. But I see the problem that this term was not used before. IMHO "... all message paths have a size that ..." does not say the right thing. For me the size of a message path describes the number of nodes or links in the path, and not the size of the message which is represented by the path. An alternative could be: "Prove that all message paths lead to a message with a size that is a multiple of 8 bit".

Reply:
> An alternative could be: "Prove that all message paths lead to a message with a size that is a multiple of 8 bit".

That would work for me.

Reply:
Changed in 149052e.